cli/tests/init.bats

```diff
@@ -172,12 +172,6 @@ EOF
   assert_line "run/"
 }
 
-@test "'flox init' injects current system" {
```

Comment:
>  fix: remove test that's no longe relevant 

(non-blocking) Can you describe why?

Reply:
This test ensures that the current system is injected into the manifest, but we now inject all 4 systems by default